test/IkuToken.js

```diff
@@ -1,6 +1,7 @@
 import decodeLogs from './helpers/decodeLogs';
 
 const IkuToken = artifacts.require('IkuToken');
+const IkuToken_v2 = artifacts.require('IkuToken_v2');
```

Comment:
What's the point of requiring IkuToken_v2 here if you're not doing anything with it?

Reply:
nice catch! I've removed it.